.github/workflows/test.yml

```diff
@@ -28,6 +39,31 @@ jobs:
             echo "Run 'go mod tidy' and push it"
             exit 1
           fi
+      - name: Install plugin from repository
+        run: |
+          trivy plugin install github.com/dramf/md
+          trivy plugin info md
+          rm -rf ~/.trivy/plugins/md
+
+      - name: Install plugin from release
```

Comment:
I thought again, testing the version from the release is not right, because we will always look at the same version (and to change we will have to update this file)
I think it is worth removing this step